app/models/page.rb

```diff
@@ -127,6 +127,16 @@ def campaign_action_count
                                end
   end
 
+  def pledger?
+    @fundraiser ||= Plugins::Fundraiser.find_by(page_id: id)
+
+    @fundraiser.try(:pledge) || false
```

Comment:
I'd use `try!` instead of `try` (or depending on the target ruby version the safe navigation `&` for better error messages)